splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/pages/Input/InputPage.jsx

```diff
@@ -215,7 +215,7 @@ function InputPage() {
                         page={PAGE_INPUT}
                         handleOpenPageStyleDialog={handleOpenPageStyleDialog}
                     />
-                    <ToastMessages />
+                    <ToastMessages position="top-right" />
```

Comment:
Please confirm: this would change all the messages location right?

Reply:
Yes, it will